gst/nnstreamer/nnstreamer_plugin_api_util_impl.c

```diff
@@ -1400,21 +1398,24 @@ gst_tensor_meta_info_init (GstTensorMetaInfo * meta)
  * @param[in] meta tensor meta structure
  * @param[out] major pointer to get the major version number
  * @param[out] minor pointer to get the minor version number
+ * @return TRUE if successfully get the version
  */
-void
+gboolean
 gst_tensor_meta_info_get_version (GstTensorMetaInfo * meta,
     guint * major, guint * minor)
 {
-  g_return_if_fail (meta != NULL);
+  g_return_val_if_fail (meta != NULL, FALSE);
 
   if (!GST_TENSOR_META_IS_VALID (meta))
-    return;
+    return FALSE;
```

Comment:
I think we can reduce the code as follows

g_return_val_if_fail (meta != NULL && GST_TENSOR_META_IS_VALID (meta), FALSE);

Reply:
thanks, updated :)